package.json

```diff
@@ -67,6 +67,7 @@
     "jquery.scrollto": "^2.1.2",
     "js-cookie": "^2.2.1",
     "patch-package": "^6.2.2",
+    "playwright": "^1.36.2",
```

Comment:
I believe the packages.json can be setup to indicate certain packages are dev/ci only so they don't get installed in the deploy node image?